api/common.go

```diff
@@ -42,6 +42,9 @@ func Init(mux *http.ServeMux) error {
 	mux.HandleFunc("POST /account/login", handleAccountLogin)
 	mux.HandleFunc("POST /account/changepw", handleAccountChangePW)
 	mux.HandleFunc("GET /account/logout", handleAccountLogout)
+	mux.HandleFunc("POST /account/addfriend", handleAddFriend)
```

Comment:
nit: the proper "rest" approach is to use the same resource with different methods:

```
POST /account/friends
DELETE /account/friends/{uuid}
GET /account/friends?status=online
```

but do as you like

Reply:
Fair point, I changed the add friend and remove friend.
On remove I made the username parameter as a query param, it seems that to have dynamic routing we would have to use another lib, and I'd rather use query params than doing it manually to not add too much parsing code when not really needed.
I like having the friends online stat separate as we're only retrieving a number, not the full friend object (which would be more suitable for a GET /friends)